packages/docusaurus/bin/docusaurus.js

```diff
@@ -94,13 +94,23 @@ cli
     '--no-minify',
     'Build website without minimizing JS bundles (default: false)',
   )
-  .action((siteDir = '.', {bundleAnalyzer, outDir, minify}) => {
-    wrapCommand(build)(path.resolve(siteDir), {
-      bundleAnalyzer,
-      outDir,
-      minify,
-    });
-  });
+  .option(
+    '--use-old-css-minifier',
```

Comment:
Do we want to keep this option over time, or is it just temporary due to the risk of merging this feature?

If this is temporary, maybe a hidden env variable that we'll give only in issues would be better than a documented option?

Reply:
Rather, it is a workaround so that in the event of broken CSS, our users can use the newer version of Docusaurus instead of rolling back to the previous one.
I do not know exactly how best to do this, perhaps, as you wrote below, expand the `noMifiny` option with the string types (simple and advanced), i.e. `yarn build --no-minify` and `yarn build --minify=simple|advanced`. WDYT?
In any case, we definitely need to document new behavior.

Reply:
On the contrary, if this is an option that only makes sense to remove the risk of merging this PR,  I'd rather not document it and have an escape hatch env variable, than commit to maintain 2 modes on the long term.

I'd like to avoid ending up with simple | advanced | more-advanced | more-advanced-v2 modification option on the long term. If  there's no interest keeping the "simple" mode  then we  should make sure that removing it in a few months would not be considered as a breaking change

Reply:
Yes, you are right, but I insist on documenting this feature so people know how they could go back to the previous CSS minification while we improving it (in case they encounter broken CSS). (We can specify that this is a temporary env variable)?

Reply:
ok to document this a bit